src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/EdgeCasesTests.cs

```diff
@@ -105,6 +105,7 @@ public static void ZeroLengthContent_RoundTrip()
                 }
                 catch (CryptographicException) when (PlatformDetection.IsFullFramework) // Expected on full FX
                 {
+                    return;
```

Comment:
This change is correct, but as a more general point I dislike tests that obscure the platform differences within the code like this. It makes it more difficult to find platform inconsistencies in the future and circumvents the justification field of the `SkipOnTargetFramework` tag.

In this test in particular, it would be nice for the test to have a reason *why* the CryptoException is expected on netfx but not netcoreapp.